online-inference/fastertransformer/download-weights-job.yml

```diff
@@ -0,0 +1,310 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: gptj-download
+spec:
+  template:
+    spec:
+      containers:
+      - name: gptj-model-downloader
+        image: nvcr.io/nvidia/tritonserver:22.05-py3
+        imagePullPolicy: IfNotPresent
+        command: 
+        - /bin/sh
```

Comment:
Just preference, but I'd put this in a script and call that instead of putting it all here.

Reply:
As discussed, we want to leave configuration out of the base docker image: https://github.com/coreweave/utility-images/pull/35#discussion_r940664136